packages/angular/src/generators/application/lib/convert-to-standalone-app.ts

```diff
@@ -45,21 +45,15 @@ function updateMainEntrypoint(
 
 const standaloneComponentMainContents = (
   routerModuleSetup
-) => `import { enableProdMode } from '@angular/core';
-import { bootstrapApplication } from '@angular/platform-browser';${
+) => `import { bootstrapApplication } from '@angular/platform-browser';${
   routerModuleSetup
     ? `
 import { provideRouter, withEnabledBlockingInitialNavigation } from '@angular/router'`
     : ``
 };
 import { AppComponent } from './app/app.component';
-import { environment } from './environments/environment';
 ${routerModuleSetup ? `import { appRoutes } from './app/app.routes';` : ''}
 
-if (environment.production) {
-  enableProdMode();
```

Comment:
What calls this now? Does anything call this? Is it not needed anymore?

Reply:
No longer required